rest-server/src/templates/dockerContainerScript.mustache

```diff
@@ -52,7 +52,25 @@ export PAI_JOB_TASK_ROLE_COUNT={{{ taskRolesNumber }}}
 export PAI_JOB_TASK_ROLE_LIST={{{ taskRoleList }}}
 export PAI_KILL_ALL_ON_COMPLETED_TASK_NUM={{{ jobData.killAllOnCompletedTaskNumber }}}
 
-export PAI_CONTAINER_HOST_PORT=$(( (RANDOM % 55535) + 10001 ))
+# generate container host port
+PAI_CONTAINER_HOST_PORT=$(( (RANDOM % 55535) + 10001 ))
```

Comment:
Can you write `PAI_CONTAINER_HOST_PORT=$(( (RANDOM % 55535) + 10001 ))` once? For example:
```
while true
  CONTAINER_HOST_PORT=...
  if not lsof -i:${PAI_CONTAINER_SSH_PORT}
    break
  else
    sleep 1
```
In this way, we don't have to change two lines if we need to modify the port range in the future.